sr_port_cm/gtcmtr_reversequery.c

```diff
@@ -0,0 +1,99 @@
+/****************************************************************
+ *								*
+ *	Copyright 2001, 2013 Fidelity Information Services, Inc	*
```

Comment:
On any new module where you need to carry the FIS copyright, it would be best to use the newer form of copyright that looks like ours instead of this older truncated version.

Reply:
I don't know if YottaDB can change the copyright lines added by FIS which is why I don't touch those usually.

Reply:
We should probably leave the FIS copyrights untouched, and just insert ours.